.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: elixir
 elixir:
 - 1.10.4
+- 1.11.0
```

Comment:
Travis-CI config is confusing, by setting it up this way we are actually running **4 builds**:

https://github.com/dwyl/fields/pull/78/checks?check_run_id=1321213543

![image](https://user-images.githubusercontent.com/194400/97497200-0b2e3880-1962-11eb-929f-b50dabb64378.png)
And it appears that two of them fail due to incompatibility of OTP/Elixir Versions. 🙄 

In order to test against specific version of Elixir+OTP we need a `Matrix` e.g:
https://github.com/phoenixframework/phoenix_html/blob/3585037f78a93f28a34805e758c9fdc9ba03049e/.travis.yml#L2-L7

e.g:
```
matrix:
  include:
    - elixir: 1.10.4
      otp_release: 22.3.4.12
    - elixir: 1.11.0
      otp_release: 23.0.4
```